ci/main.py

```diff
@@ -118,14 +126,11 @@ def upload(name, version, channel):
         config.bldpkgs_dir,
         '{0}-{1}*.tar.bz2'.format(name, version))
     built = glob.glob(built_glob)[0]
-    upload_cmd = 'anaconda -t {token} upload -u {channel} {built}'
+    upload_cmd = 'anaconda -t {token} upload -u %s %s' % (channel, built)
     # Do not show decrypted token!
     log.info('Uploading: {0}'.format(upload_cmd))
     check_call(
-        upload_cmd.format(
-            token=os.environ['ANACONDA_TOKEN'],
-            built=built,
-            channel=channel),
```

Comment:
I changed it conveniently while I was at it. you think it is gilding the
lily?

On Fri, Nov 6, 2015 at 9:37 AM, Jorge Cañardo Alastuey <
notifications@github.com> wrote:

> In ci/main.py
> <https://github.com/biocore/conda-recipes/pull/31#discussion_r44164503>:
>
> >      # Do not show decrypted token!
> >      log.info('Uploading: {0}'.format(upload_cmd))
> >      check_call(
> > -        upload_cmd.format(
> > -            token=os.environ['ANACONDA_TOKEN'],
> > -            built=built,
> > -            channel=channel),
>
> Can you keep the command formatting in the same call? Thanks!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/conda-recipes/pull/31/files#r44164503>.
>